store/src/sql/mod.rs

```diff
@@ -29,15 +31,16 @@ pub struct SqlitePool {
 impl SqlitePool {
     /// Connect to the sqlite database at the given path. Creates the database if it does not exist.
     /// Uses WAL journal mode.
-    pub async fn connect(path: impl AsRef<Path>) -> anyhow::Result<Self> {
+    pub async fn connect(path: impl AsRef<Path>, migrate: bool) -> anyhow::Result<Self> {
```

Comment:
Can we enum the migrate flag? Or split to a connect and connect_with_migrate?

Reply:
Added an enum.. wrapped the memory pool in its own function so now we only call this from a few places in the binary rather than all over in test code